packages/fxa-content-server/app/scripts/views/mixins/service-mixin.js

```diff
@@ -32,7 +33,11 @@ export default {
       service,
       serviceName: subscriptionProductName || serviceName,
       serviceLogo,
-      isSync: this.relier.isSync(),
+      // relier.isSync() doesn't appear to get set from deeplink in mobile FF
+      // so also check serviceName, see #13425
+      isSync:
+        this.relier.isSync() ||
+        serviceName === Constants.RELIER_SYNC_SERVICE_NAME,
```

Comment:
This was a bit of a weird one. I looked around at `isSync` in the codebase and found a comment in `reliers/base.js` that says, "Check if the relier is Sync for Firefox Desktop". Though, in most files, we're checking exclusively for `this.relier.isSync()`, and then in other places we check for `service === 'sync'` or we check against the `serviceName`.

I can see in the screenshot that QA provided that clearly, the `serviceName` is getting passed in and rendering correctly, so this _should_ fix that ticket. I originally changed the check to _just_ check for the serviceName, but it caused tests in other files to fail, and because Backbone I'm not sure if this is overwriting other `isSync` checks that are referencing `this.relier.isSync()`, like [this one](https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/app/scripts/views/ready.js#L117) in `ready.js`. Having both checks seems safest so that's what I went with.

Reply:
Ahh, things suddenly make sense and that's much cleaner so thanks. 😁

I updated my PR. I checked against `serviceName` since it was less logic as opposed to the permission scope but maybe you didn't mean that anyway with "value on that scope".